app/views/sections/show.html.erb

```diff
@@ -1,12 +1,13 @@
 <% content_for :title, goods_nomenclature_title(@section) %>
+
 <% content_for :head do %>
   <meta name="description" content="<%= @section %>">
   <link rel='alternate' type='application/json' href='<%= section_url(@section, format: :json) %>' title='Section information page in JSON format' />
 <% end %>
 
 <% content_for :bodyClasses, "page--wide" %>
 
-<h2 class="govuk-heading-m">Choose the chapter that best matches your goods</h2>
+<p>Section <span class="govuk-body govuk-!-font-weight-bold"><%= @section.numeral %></span> contains <span class="govuk-body govuk-!-font-weight-bold"><%= @section.chapters_count %></span> chapters. Choose the chapter that best matches your goods.</p>
```

Comment:
Does this thist need `govuk-body` on it - `<p>` already has the `govuk-body` class applied automatically?

A `<strong>` tag would give semantic representation as well as visual implementation which may be useful for screen readers?